doc/cli.markdown

```diff
@@ -983,8 +984,10 @@ Examples:
 
 	$ balena env add TERM --application MyApp
 	$ balena env add EDITOR vim --application MyApp
+	$ balena env add EDITOR vim --application MyApp,MyApp2
 	$ balena env add EDITOR vim --application MyApp --service MyService
```

Comment:
What about allowing multiple services to be listed as well? -
```
$ balena env add EDITOR vim --application MyApp,MyApp2 --service MyService,MyService2
```

Reply:
I think this makes sense. I'll add it in.